code/modules/events/traders.dm

```diff
@@ -82,3 +121,110 @@ GLOBAL_LIST_INIT(unused_trade_stations, list("sol"))
 	objs += new /datum/objective/trade/credits
 
 	return objs
+
+//Datums that handle the various announcements, species, outfits, and item lists.
+/datum/traders
+	/// What faction the trader is
+	var/trader_type
+	/// Where the traders originate from
+	var/trader_location
+	/// What specific station the traders came from
+	var/dock_site
+	/// What species the traders consist of
+	var/trader_species
+	/// What outfit do the traders spawn with
+	var/trader_outfit
+	/// What standard faction gear do they start with
+	var/trader_minor_special
+	/// What big ticket faction gear do they start with
+	var/trader_major_special
+
+/datum/traders/sol
+	trader_type = "Trans-Solar Federation"
+	trader_location = "Kayani Station"
+	dock_site = "Kayani Station"
+	trader_species = /mob/living/carbon/human
+	trader_outfit = /datum/outfit/admin/trader/sol
+	trader_minor_special = /obj/effect/spawner/lootdrop/trade_sol/federation_minor
+	trader_major_special = /obj/effect/spawner/lootdrop/trade_sol/federation_major
+
+/datum/traders/cyber
+	trader_type = "Robomoon Industries" //Cybersun, but *legally distinct*
```

Comment:
This is a little bit on the nose imo, and the only place this appears is to dchat and to admins. I say just call it Cybersun